scheds/rust/scx_layered/src/main.rs

```diff
@@ -2282,6 +2298,26 @@ impl<'a> Scheduler<'a> {
         Ok(())
     }
 
+    fn refresh_idle_qos(&mut self) -> Result<()> {
+        if !self.idle_qos_enabled {
+            return Ok(());
+        }
+
+        let mut cpu_idle_qos = vec![0; *NR_CPU_IDS];
+        for layer in self.layers.iter() {
+            let idle_resume_us = layer.kind.common().idle_resume_us.unwrap_or(0) as i32;
+            for cpu in layer.cpus.iter() {
+                cpu_idle_qos[cpu] = idle_resume_us;
+            }
+        }
+
+        for (cpu, idle_resume_usec) in cpu_idle_qos.iter().enumerate() {
```

Comment:
Here, we probably should iterate `topo.all_cpus` but that's broken everywhere, so let's worry about that later.